databricks/koalas/frame.py

```diff
@@ -8337,11 +8337,25 @@ def query(self, expr, inplace=False):
 
         .. note::
             * MultiIndex columns are not supported.
-            * If you want to use MultiIndex columns, you can just workaround
-              by using :meth:`DataFrame.apply` although it is less performant.
+            * Using variables in the environment with `@` is not supported.
             * Internal columns that starting with __. are able to access,
               however, they are not supposed to be accessed.
             * This delegates to Spark SQL so the syntax follows Spark SQL
+            * If you want to use MultiIndex columns, or variables in the environment,
+              you can just workaround by using :meth:`DataFrame.map_in_pandas`
+              although it is less performant. See the example below.
+
+            >>> df = ks.DataFrame([(1, 2), (3, 4), (5, 6)], columns=['A', 'B'])
```

Comment:
I think it should have four spaces more. Does it render okay in the doc?

Reply:
yeah i checked, and it was looks fine to me.

<img width="505" alt="스크린샷 2020-02-14 오후 12 55 15" src="https://user-images.githubusercontent.com/44108233/74500125-582fc300-4f29-11ea-8e9b-3b8992c0e95c.png">

but i just tested with 4 more indent as you mentioned, it looks better.

<img width="487" alt="스크린샷 2020-02-14 오후 12 58 57" src="https://user-images.githubusercontent.com/44108233/74500289-c8d6df80-4f29-11ea-9f63-079b0f1e16ac.png">

Reply:
fixed! thanks for the comment :)